android/build.gradle

```diff
@@ -36,7 +36,8 @@ android {
 dependencies {
     implementation 'com.facebook.react:react-native:+' // From node_modules
 
-    implementation 'com.github.zoom-us-community:jitpack-zoom-us:5.16.2.16555'
+    // implementation 'com.github.zoom-us-community:jitpack-zoom-us:5.16.2.16555'
+    implementation 'com.github.chirag-codealchemy:zoom:5.17.1.18530'
```

Comment:
5.17 is already available in the zoom-us-community jitpack
https://github.com/zoom-us-community/jitpack-zoom-us